zerberus/sfz.cpp

```diff
@@ -89,6 +89,9 @@ struct SfzRegion {
       LoopMode loop_mode;
       OffMode off_mode;
       std::map<int, double> gain_oncc;
+      float delay; //seconds
+      int pan;    // [-100, 100]
+      int offset; // [0, 4Gb] or [0, 4294967296]
```

Comment:
Need `unsigned long` here

Reply:
You are right! Even unsigned int should be enough if max offset is 4Gb-1bit :)  BTW, unsigned needs additional logic in reading values. Thank you for sharp eye overview ;)

Reply:
`int`is only guaranteed to be at least 16 bit wide, only long is guaranteed to be at least 32 bit wide

Reply:
to be really save use `uint32_t`

Reply:
`int` is only guaranteed to be at least 16 bit wide, only long is guaranteed to be at least 32 bit wide.
To be really save use `uint32_t` although these are optional in C++11, while `uint_least32_t` is not. See http://en.cppreference.com/w/cpp/types/integer.

Reply:
If we really need 4294967296, we need long long